maths/gcd_of_n_numbers.py

```diff
@@ -11,21 +11,21 @@ def get_factors(
 ) -> Counter:
     """
     this is a recursive function for get all factors of number
-    >>> get_factors(45)
+    # get_factors(45)
     Counter({3: 2, 5: 1})
-    >>> get_factors(2520)
+    # get_factors(2520)
```

Comment:
Please read `CONTRIBUTING.md`.

`>>>` is different than `#` so please **revert** theses changes.

https://docs.python.org/3/library/doctest.html